README.md

```diff
@@ -0,0 +1,187 @@
+<<<<<<< HEAD
+=======
+# Capstone-project
+>>>>>>> 13f268006e8657165599c557ac2cbf78a0538b60
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+<<<<<<< HEAD
+- [📖 My Portfolio ](#-my-portfolio-)
+=======
+- [📖 Caspstone ](#-capstone-project-)
+>>>>>>> 13f268006e8657165599c557ac2cbf78a0538b60
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
```

Comment:
- [ ] `Required`
- [ ] Good try 👍 
- [ ] However, your live link is not working 😭 
- [ ] kindly update it to working or deploy if not deployed yet in accordance with the requirements. 🥂